Tools/TestWebKitAPI/Tests/WebKitCocoa/SmartLists.mm

```diff
@@ -342,6 +342,17 @@ static void runTest(NSString *input, NSString *expectedHTML, NSString *expectedS
     runTest(@"+42. Hello", expectedPlusPrefixedHTML.createNSString().get(), @"//body/text()", @"+42. Hello".length);
 }
 
+TEST(SmartLists, InsertingSpaceAfterLargeNumberDoesNotGenerateOrderedList)
+{
+    static constexpr auto expectedHTML = R"""(
+    <body>1000000000000000. hi</body>
+    )"""_s;
+
+    static constexpr NSString* input = @"1000000000000000. hi";
```

Comment:
I dont think the constexpr here is actually doing anything? I dont think you can have a constexpr NSString * afaik but I could be wrong